src/features/Home/Calendar/CalendarModal/index.tsx

```diff
@@ -1,34 +1,34 @@
 import * as s from './style.css';
 import Dropdown from '../Dropdown';
 import useEventState from './calendarModal.hook';
-import {
-  CalendarEvent,
-  ScheduleCategory,
-  TodoPriority,
-} from 'entities/calendar/type';
+import { CalendarEvent } from 'entities/calendar/type';
+import { categorys } from 'entities/calendar/model';
 
 interface CalendarModalProps {
   onClose: () => void;
   event?: CalendarEvent;
 }
 
 const CalendarModal = ({ onClose, event }: CalendarModalProps) => {
-  const { state, updateState, switchEventType, isInitial } = useEventState({
-    event,
-  });
+  const { state, updateState, switchEventType, isInitial, createEvent } =
+    useEventState({
+      event,
+    });
 
-  const handleChangeRepeat = (value: string) =>
-    updateState({ selectedRepeat: value });
+  const handleChangeRepeat = (id: number) =>
+    updateState({ selectedRepeatId: id });
 
-  const handleChangeCategory = (value: string) =>
-    updateState({
-      selectedCategory: value as ScheduleCategory,
-    });
+  const handleChangeCategory = (id: number) =>
+    updateState({ selectedCategoryId: id });
 
-  const handleChangePriority = (value: string) =>
-    updateState({
-      selectedPriority: value as TodoPriority,
-    });
+  const handleChangePriority = (id: number) =>
+    updateState({ selectedPriorityId: id });
+
+  const handleChangeTitle = (e: React.ChangeEvent<HTMLInputElement>) =>
+    updateState({ title: e.target.value });
+
+  const handleChangeMemo = (e: React.ChangeEvent<HTMLInputElement>) =>
+    updateState({ memo: e.target.value });
```

Comment:
```suggestion
const handleChangeInputs = (key: 'title' | 'content', value: string) => {
    updateState({ [key]: value });
  };
```
함수를 하나도 간추릴 수 있을 것 같네요.